t3f/ops.py

```diff
@@ -793,29 +793,39 @@ def frobenius_norm_squared(tt, differentiable=False):
   """Frobenius norm squared of a TensorTrain (sum of squares of all elements).
 
   Args:
-    tt: `TensorTrain` object
+    tt: `TensorTrain` or `TensorTrainBatch` object
     differentiable: bool, whether to use a differentiable implementation
       or a fast and stable implementation based on QR decomposition.
 
   Returns
-    a number
-    sum of squares of all elements in `tt`
+    a number which is the Frobenius norm squared of `tt`, if it is `TensorTrain`
+    OR
+    a Tensor of size tt.batch_size, consisting of the Frobenius norms squared of
+    each TensorTrain in `tt`, if it is `TensorTrainBatch`
   """
   if differentiable:
+    if hasattr(tt, 'batch_size'):
+        bs_str = 'n'
+    else:
+        bs_str = ''
     if tt.is_tt_matrix():
-      running_prod = tf.einsum('aijb,cijd->bd', tt.tt_cores[0], tt.tt_cores[0])
+      running_prod = tf.einsum('{0}aijb,{0}cijd->{0}bd'.format(bs_str),
+                                tt.tt_cores[0], tt.tt_cores[0])
```

Comment:
Sorry, but PEP-8 again :)
Remove a space here: the second line should start right after the bracket in the previous line in this case.